src/bioregistry/data/bioregistry.json

```diff
@@ -70498,6 +70498,56 @@
       }
     ]
   },
+  "mirgenedb": {
+    "contact": {
+      "email": "BastianFromm@gmail.com",
+      "name": "Bastian Fromm",
+      "orcid": "0000-0003-0352-3037"
+    },
+    "contributor": {
+      "email": "palayoor.n@northeastern.edu",
+      "github": "nalikapalayoor",
+      "name": "Nalika Palayoor",
+      "orcid": "0009-0008-8406-631X"
+    },
+    "description": "mircoRNA Gene Database is a collection of animal microRNA genes. This database allows researchers to analyze mircoRNA properties like secondary structure and features in a phylogenetic context.",
+    "example": "hsa/Let-7-P1b",
```

Comment:
This is not a valid ID, it could be e.g., Hsa-Let-7-P2a3 as in my previous comment